app/ipc/paths.ts

```diff
@@ -16,6 +15,10 @@ class Paths {
   root() {
     return app.getAppPath()
   }
+
```

Comment:
nit: consistent new lines in between class members.

Reply:
I suppose but a new line above root since that seems to be what you're doing in the rest of this pr.